flink/v1.20/flink/src/test/java/org/apache/iceberg/flink/sink/TestFlinkIcebergSink.java

```diff
@@ -52,21 +52,36 @@ public class TestFlinkIcebergSink extends TestFlinkIcebergSinkBase {
   @Parameter(index = 2)
   private boolean partitioned;
 
-  @Parameters(name = "format={0}, parallelism = {1}, partitioned = {2}")
+  @Parameter(index = 3)
+  private boolean useV2Sink;
+
+  @Parameters(name = "format={0}, parallelism = {1}, partitioned = {2}, useV2Sink = {3}")
   public static Object[][] parameters() {
     return new Object[][] {
-      {FileFormat.AVRO, 1, true},
-      {FileFormat.AVRO, 1, false},
-      {FileFormat.AVRO, 2, true},
-      {FileFormat.AVRO, 2, false},
-      {FileFormat.ORC, 1, true},
-      {FileFormat.ORC, 1, false},
-      {FileFormat.ORC, 2, true},
-      {FileFormat.ORC, 2, false},
-      {FileFormat.PARQUET, 1, true},
-      {FileFormat.PARQUET, 1, false},
-      {FileFormat.PARQUET, 2, true},
-      {FileFormat.PARQUET, 2, false}
+      {FileFormat.AVRO, 1, true, false},
```

Comment:
We don't need the whole testing matrix here.
I usually:
- Test everything with the new feature
- Run some choice tests that makes sure the old feature is intact - Maybe can run 1 Avro, 1 ORC test and keep all the PARQUET for the old Sink?

Reply:
Ok, reduced